dune

```diff
@@ -12,17 +12,7 @@
  (package hacl-star)
   (mld_files index))
 
-(executable
- (name gen_dune_rules)
- (modules Gen_dune_rules))
-
 (rule
- (mode promote)
-(action
- (with-stdout-to dune.inc
- (run ./gen_dune_rules.exe))
-))
-
-(include dune.inc)
-
-(data_only_dirs raw)
+  (deps raw/config.h)
+  (target config.h)
+  (action (copy %{deps} %{target})))
```

Comment:
I think that it can be replaced by `(copy_files raw/config.h)`. Or if you want to keep it as a rule, `(rule (copy raw/config.h config.h))` is enough.

Reply:
Yes, that is nicer and works very well.